api/src/main/resources/workbench.yaml

```diff
@@ -1250,6 +1296,41 @@ definitions:
         operator: "BETWEEN"
         operands: ["18", "35"]
 
+  ParticipantListResponse:
+    type: object
+    required:
+      - items
+    properties:
+      items:
+        type: "array"
+        items:
+          $ref: "#/definitions/Participant"
+
+  Participant:
+    type: object
+    required:
+      - cohortId
+      - cdrVersionId
+      - participantId
+      - status
+    properties:
+      cohortId:
+        type: integer
+        format: int64
+      cdrVersionId:
+        type: integer
+        format: int64
+      participantId:
+        type: integer
+        format: int64
+      status:
+        $ref: "#/definitions/CohortStatus"
+
+  CohortStatus:
+    type: string
+    description: possible values indicating whether participants are in or out of the cohort
+    enum: [Not Reviewed, Included, Excluded, Needs Further Review]
```

Comment:
I slightly prefer NOT_REVIEWED, INCLUDED, etc. here... that gets across that these are enum values rather than display strings, and allows us to look up the enum values exactly matching the values used in the API.

Reply:
changed to reflect enum values